action.yml

```diff
@@ -46,13 +46,41 @@ runs:
             const diffInSeconds = (currentRunDurationInMillis - latestMasterRunDurationInMillis) / 1000
             const percentageDiff = ((1 - (currentRunDurationInMillis/latestMasterRunDurationInMillis)) * 100).toFixed(2)
             const outcome = (diffInSeconds > 0) ? "an increase" : "a decrease"
-         
+
             outputMessage =  '🕒 Workflow \"' + context.workflow + '\" took ' +  (currentRunDurationInMillis / 1000) + 's which is ' + outcome + ' with ' + Math.abs(diffInSeconds) + 's (' + Math.abs(percentageDiff) + '%) compared to latest run on master/main.'
           }
 
-          github.rest.issues.createComment({
-            issue_number: context.issue.number,
+          var commentData = {
             owner: context.repo.owner,
             repo: context.repo.repo,
-            body: outputMessage
+            issue_number: context.issue.number,
+          };
+
+          var comments = github.rest.issues.listComments(commentData);
```

Comment:
Is there any reason for using `var` instead of `let` and `const`?

Reply:
Just trying to match the coding style in the rest of the document. Happy to adjust!

Reply:
Makes total sense! Would be much appreciated if you could adjust 🙌🏼